e2e/test/config/TestConfiguration.Provisioning.cs

```diff
@@ -20,11 +20,9 @@ public static partial class Provisioning
             public static string IdScope => GetValue("DPS_IDSCOPE");
 
             // To generate use Powershell: [System.Convert]::ToBase64String( (Get-Content .\certificate.pfx -Encoding Byte) )
-            public static X509Certificate2 GetIndividualEnrollmentCertificate()
```

Comment:
X509 self-signed certificates are being generated on-demand, so this can be removed